improver/cli/threshold.py

```diff
@@ -129,15 +127,15 @@ def process(
         thresholds = []
         fuzzy_bounds = []
         for key in threshold_config.keys():
-            thresholds.append(np.float32(key))
+            thresholds.append(float(key))
             # If the first threshold has no bounds, fuzzy_bounds is
             # set to None and subsequent bounds checks are skipped
             if threshold_config[key] == "None":
                 fuzzy_bounds = None
                 continue
             fuzzy_bounds.append(tuple(threshold_config[key]))
     else:
-        thresholds = [np.float32(x) for x in threshold_values]
+        thresholds = [float(x) for x in threshold_values]
```

Comment:
Similarly a comment would be useful here e.g. `Ensure thresholds are float64 to avoid rounding errors during possible unit conversion.`.